packages/admin-panel/src/api/TupaiaApi.js

```diff
@@ -77,9 +79,21 @@ export class TupaiaApi {
     };
   }
 
+  async checkIfAuthorized(response) {
+    // Unauthorized
+    if (response.status === 401) {
```

Comment:
Initially I was wondering whether it is too aggressive to always log out users when there is a 401 but after thinking about it for a while I agree that it makes good sense.